modules/ssl/ssl_engine_kernel.c

```diff
@@ -2447,7 +2499,17 @@ int ssl_callback_ClientHello(SSL *ssl, int *al, void *arg)
     const unsigned char *pos;
     size_t len, remaining;
     (void)arg;
- 
+
+#ifdef HAVE_OPENSSL_ECH
+    if (SSL_client_hello_get0_ext(ssl, TLSEXT_TYPE_ech, &pos, &remaining)) {
+        ap_log_cerror(APLOG_MARK, APLOG_INFO, 0, c, APLOGNO(10540)
+                      "there is an ECH extension");
+    } else {
+        ap_log_cerror(APLOG_MARK, APLOG_INFO, 0, c, APLOGNO(10541)
+                      "there is NO ECH extension");
+    }
+#endif
```

Comment:
Not sure this is needed since ssl_callback_ECH() will log the status anyway?
And I don't think we want this to be logged at APLOG_INFO for every connection, APLOG_DEBUG/TRACEn would be enough IMHO.

Reply:
yeah, that's fair - I guess I added that while figuring stuff out;-)

it's removed in the push below